container-setup/install/install-kube_ps1.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+if [ "$I_AM_IN_CONTAINER" != "I-am-in-container" ]; then
+  echo "must be run in container";
+  exit 1;
+fi
+
+echo "in container";
+
+mkdir /usr/local/kube_ps1;
+pushd /usr/local/kube_ps1;
+wget -q https://raw.githubusercontent.com/drewandersonnz/kube-ps1/master/kube-ps1.sh;
```

Comment:
As above, this fork is a safety-net, we are not using upstream directly to be able to review any code additions. This could be vendored, or cloned to openshift org, or some other thing. But some link through to finally upstream would be good for recognition of that projects contributions, and for being able to stay in sync.